### PR TITLE
Search Form Style - Improved

### PR DESCRIPTION
**Pull request by iamibrahimriaz**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Security fix
- [x] Improvement
- [ ] New Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Text changes
- [ ] Other... Please describe:

## Description

1. Markup Changed to Improve Accessibility & SEO.
2. Searchform Style Improvement.
3. Style Improvement.

How to reproduce the issue or how to test the changes on Searchform - 

Issue was:
image
image

Updated Style:
https://www.loom.com/share/52f1cfdcaed1404f9dc55b9f238756c4

## Checklist

- [x] My code follows the [WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)

### CODE REVIEW

**Review by devShamim — CHANGES_REQUESTED** [review]

File: templates/single/section-related_listings.php [hunk 1]
```diff
@@ -14,11 +14,11 @@
 }
 ?>
 
-<div class="directorist-related <?php echo esc_attr( $class );?>" <?php $listing->section_id( $id ); ?>>
+<section class="directorist-related <?php echo esc_attr( $class );?>" <?php $listing->section_id( $id ); ?>>
```
Comment (devShamim) [inline templates/single/section-related_listings.php:17]: 'direcotirst-related' - what?

**Review by devShamim — CHANGES_REQUESTED** [review]

File: templates/single/section-related_listings.php (hunk 1 above)
Comment (iamibrahimriaz) [reply templates/single/section-related_listings.php:17]: Fixed

File: templates/archive/advance-search-form.php [hunk 2]
```diff
@@ -13,14 +13,14 @@
 	<div class="directorist-search-form-box directorist-search-form__box">
 		<div class="directorist-advanced-filter__top">
```
[the remark this answers is not in the document]
Comment (iamibrahimriaz) [reply templates/archive/advance-search-form.php:14]: Fixed

File: templates/archive/loop-grid.php [hunk 3]
```diff
@@ -10,7 +10,7 @@
 $loop_fields = $listings->loop['card_fields']['template_data']['grid_view_with_thumbnail'];
 ?>
 
-<div class="directorist-listing-single directorist-listing-single--bg directorist-listing-card directorist-listing-has-thumb <?php echo esc_attr( $listings->loop_wrapper_class() ); ?>">
+<article class="directorist-listing-single directorist-listing-single--bg directorist-listing-card directorist-listing-has-thumb <?php echo esc_attr( $listings->loop_wrapper_class() ); ?>">
 
 	<figure class="directorist-listing-single__thumb">
```
[the remark this answers is not in the document]
Comment (iamibrahimriaz) [reply templates/archive/loop-grid.php:15]: Fixed

File: templates/archive/fields/listings_location.php [hunk 4]
```diff
@@ -2,15 +2,18 @@
...
  */
 
 if ( ! defined( 'ABSPATH' ) ) exit;
 ?>
 
-<div class="directorist-listing-card-location">
-    <?php directorist_icon( $icon );?><span class="directorist-listing-single__info--list__label"><?php $listings->print_label( $label ); ?></span>
+<li class="directorist-listing-card-location">
+    <?php directorist_icon( $icon );?>
+    <span class="directorist-listing-single__info__list__label">
```
[the remark this answers is not in the document]
Comment (iamibrahimriaz) [reply templates/archive/fields/listings_location.php]: Fixed

File: templates/archive/fields/phone2.php [hunk 5]
```diff
@@ -15,4 +15,4 @@
 );
 ?>
 
-<div class="directorist-listing-card-phone2"><?php directorist_icon( $icon ); ?><span class="directorist-listing-single__info--list__label"><?php $listings->print_label( $label ); ?></span><a href="<?php echo esc_url( Helper::phone_link( $phone_args ) ); ?>"><?php echo esc_html( $value ); ?></a></div>
\ No newline at end of file
+<li class="directorist-listing-card-phone2"><?php directorist_icon( $icon ); ?><span class="directorist-listing-single__info__list__label"><?php $listings->print_label( $label ); ?></span><a href="<?php echo esc_url( Helper::phone_link( $phone_args ) ); ?>"><?php echo esc_html( $value ); ?></a></li>
```
[the remark this answers is not in the document]
Comment (iamibrahimriaz) [reply templates/archive/fields/phone2.php]: Fixed

**Pull request merged**